### PR TITLE
feat: authentication added

### PR DESCRIPTION
**Pull request by shobit1337**
Successfully implemented Authentication with Private Routings and Links
- Login
- Signup

### CODE REVIEW

**Review by znjs — APPROVED** [review]
Clean code
Good folder structure
Good naming conventions

File: src/components/Auth/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        </Link>
+      </span>
+      <span>
+        Don't have a Gamotore Account?{' '}
+        <Link to='/signup' className='link'>
+          Sign Up
+        </Link>
+      </span>
+      <span>
+        Back to{' '}
```
Comment (ScoobyDubeyDo) [inline src/components/Auth/Login.jsx:74]: **suggestion**: avoid using `{" "}`  for extra spaces, you can wrap this text in a `span` tag and give padding to it.

**Review by ScoobyDubeyDo — APPROVED** [review]

**Review by manismk — APPROVED** [review]
Still using gamestore in auth pages change to reelflix

Rest looks good. Good folder structure

**Review by aniketxparihar — APPROVED** [review]
Good to go

**Pull request merged**